internal/cmdrpkgcopy/command.go

```diff
@@ -111,22 +112,43 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: *r.cfg.Namespace,
 		},
-		Spec: porchapi.PackageRevisionSpec{
-			PackageName:    r.target,
-			Revision:       r.revision,
-			RepositoryName: r.repository,
-			Tasks: []porchapi.Task{
-				{
-					Type: porchapi.TaskTypeEdit,
-					Edit: &r.copy,
-				},
-			},
-		},
+		Spec: *revisionSpec,
 	}
 	if err := r.client.Create(r.ctx, pr); err != nil {
 		return errors.E(op, err)
 	}
-
 	fmt.Fprintf(cmd.OutOrStdout(), "%s created", pr.Name)
 	return nil
 }
+
+func (r *runner) getPackageRevisionSpec() (*porchapi.PackageRevisionSpec, error) {
+	newScheme := runtime.NewScheme()
+	if err := porchapi.SchemeBuilder.AddToScheme(newScheme); err != nil {
+		return nil, err
+	}
+	restClient, err := porch.CreateRESTClient(r.cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	result := porchapi.PackageRevision{}
+	err = restClient.
+		Get().
+		Namespace(*r.cfg.Namespace).
+		Resource("packagerevisions").
+		Name(r.copy.Source.Name).
+		Do(context.Background()).
```

Comment:
we want to use the same context here, suggest changing to
`r.ctx`

Reply:
done! Thanks